build/Scripts/copyOutput.cmd

```diff
@@ -1,3 +1,8 @@
+@echo off
```

Comment:
Other repos have standardized on doing something like `@if not defined _echo @echo off`, as it allows people to easily debug the scripts when something fails.